README.md

```diff
@@ -12,19 +12,20 @@ $ npm install spid-passport
 
 ## Utilizzo
 ### Configurazione
-Sono necessari i parametri di configurazione per l'Identity Provider e per il Service Provider e nello specifico il costruttore prende in input due oggetti e una callback di verifica:
+Sono necessari i parametri di configurazione per l'Identity Provider e per i diversi Service Provider e nello specifico il costruttore prende in input due oggetti e una callback di verifica:
 
 ##### Service Provider: 
-- (String) `entity_id` - Nome entita che fornisce il servizio
-- (Stirng) `private_key` - Chiave privata del Service Provider (Formato PEM)
-- (String) `certificate` - Certificato Service Provider (Formato PEM)
-- (String) `assert_endpoint` - Endpoint sul quale ricevere la response dall'identity provider
+- (String) `issuer` - Nome entita che fornisce il servizio
+- (String) `privateCert` - Chiave privata del Service Provider (Formato PEM)
+- (String) `path` - Endpoint sul quale ricevere la response dall'identity provider
```

Comment:
non è chiaro se path è una url o un percorso (come nell'esempio sottostante)

Reply:
in effetti andrebbe riportata la documentazione ufficiale https://github.com/bergie/passport-saml#config-parameter-details